views/signUp.handlebars

```diff
@@ -1,22 +1,24 @@
-<h2>Sign Up</h2>
-
-<form class="form signup-form">
-  <div class="form-group">
-    <label for="username">Full Name:</label>
-    <input class="form-input" type="text" id="username" />
-  </div>
-  <div class="form-group">
-    <label for="email-signup">Email:</label>
-    <input class="form-input" type="text" id="email-signup" />
-  </div>
-  <div class="form-group">
-    <label for="password-signup">Password:</label>
-    <input class="form-input" type="password" id="password-signup" />
-  </div>
-  
-  <div class="form-group">
-    <button class="btn btn-primary" type="submit">Signup</button>
-  </div>
-</form>
+<div class="container">
+  <div class="card text-white bg-info mb-3 m-5" style="max-width: 20rem;">
+    <form class="class-header">
+      <h4 class="card-title m-2">Sign Up</h4>
+      <div class="card-body">
+        <div class="m-4">
+          <label for="username">Full Name:</label>
+          <input class="form-input" type="text" id="username" />
+        </div>
+      <div class="form-group m-4">
+        <label for="email-signup">E-mail:</label>
+        <input class="form-input" type="text" id="email-signup" />
+      </div>
+      <div class="form-group m-4">
+        <label for="password-signup">Password:</label>
+        <input class="form-input" type="password" id="password-signup" />
+      </div>
+      
+      <div class="form-group">
+        <button class="btn btn-secondary m-4" type="submit">Signup</button>
+      </div>
+    </form>
 
```

Comment:
Should no there be closing tags for the outer `divs`

Reply:
Those are needed for the child container inside of the main card body so that the button can be targeted and manipulated.

Reply:
okay